packages/next/src/lib/server-external-packages.json

```diff
@@ -11,6 +11,7 @@
   "@react-pdf/renderer",
   "@sentry/profiling-node",
   "@swc/core",
+  "@temporalio/client",
```

Comment:
This needs to be added in https://github.com/vercel/next.js/blob/canary/docs/02-app/02-api-reference/05-next-config-js/serverComponentsExternalPackages.mdx as well